Make grammar changes on Code of Conduct page (#2486)
privacyguides/privacyguides.org: docs/CODE_OF_CONDUCT.md

1. **Do not spread misinformation**

      We are creating an evidence-based educational community around information privacy and security, not an information home for conspiracy theories. For example, when making a claim that a certain piece of software is malicious or that certain telemetry data is privacy invasive, explain in detail what is collected and how it collected. Claims of this nature must be backed by technical evidence.

2. **Do not abuse our willingness to help**

>       Our community members are not free tech support. We are happy to help with specific steps for an individual's privacy journey, if they are willing to put in effort. We are not obligated to answer endless, repetitive questions about general computer problems solvable with a simple internet search. **Do not** become a [help vampire](https://slash7.com/2006/12/22/vampires).

Reviewer: ```suggestion
      Our community members are not free tech support. We are happy to help with specific steps for your privacy journey, if you are willing to put in effort. We are not obligated to answer endless, repetitive questions about general computer problems solvable with a simple internet search. **Do not** become a [help vampire](https://slash7.com/2006/12/22/vampires).
```

- Any other conduct which would reasonably be considered inappropriate in a professional setting.
- Public and/or private harassment of any kind.

> - Publishing others' private information, such as a physical address and/or an email address, without a person's explicit permission.

Reviewer: ```suggestion
- Publishing others' private information, such as a physical address and/or an email address, without their explicit permission.
```
